core/tx_pool.go

```diff
@@ -1946,7 +1946,6 @@ func (pool *TxPool) promoteExecutables(accounts []common.InternalAddress) []*typ
 		queuedRateLimitMeter.Add(float64(len(caps)))
 		// Mark all the items dropped as removed
 		pool.priced.Removed(len(forwards) + len(drops) + len(caps))
-		queuedRateLimitMeter.Sub(float64(len(forwards) + len(drops) + len(caps)))
```

Comment:
are we deleting this? is it calculated somewhere else ?

Reply:
Yes, we are deleting this. This line is the bug, which forces the quedRateLimitMeter to go negative.

The ask was for me to restore the queuedRateLimitMeter to what Geth has, and this line was the only difference.